Makefile

```diff
@@ -42,21 +42,21 @@ lint:
 
 test: test-cli test-fpm test-http
 
-test-nts: ./tmp/build-nts.tags
-	xargs -I % ./test-nts.sh % < ./tmp/build-nts.tags
+test-nts: ./docker-image/build-nts.tags
+	xargs -I % ./test-nts.sh % < ./docker-image/build-nts.tags
 
-test-zts: ./tmp/build-zts.tags
-	xargs -I % ./test-zts.sh % < ./tmp/build-zts.tags
+test-zts: ./docker-image/build-zts.tags
+	xargs -I % ./test-zts.sh % < ./docker-image/build-zts.tags
 
 scan-vulnerability:
 	docker-compose -f test/security/docker-compose.yml -p clair-ci up -d
 	RETRIES=0 && while ! wget -T 10 -q -O /dev/null http://localhost:6060/v1/namespaces ; do sleep 1 ; echo -n "." ; if [ $${RETRIES} -eq 10 ] ; then echo " Timeout, aborting." ; exit 1 ; fi ; RETRIES=$$(($${RETRIES}+1)) ; done
-	cat ./tmp/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./tmp/clair/%.json" -l ./tmp/clair/clair.log % || echo "% is vulnerable"'
+	cat ./docker-image/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./docker-imageclair/%.json" -l ./clair/clair.log % || echo "% is vulnerable"'
 	docker-compose -f test/security/docker-compose.yml -p clair-ci down
 
 ci-scan-vulnerability:
 	docker-compose -f test/security/docker-compose.yml -p clair-ci up -d
 	RETRIES=0 && while ! wget -T 10 -q -O /dev/null http://localhost:6060/v1/namespaces ; do sleep 1 ; echo -n "." ; if [ $${RETRIES} -eq 10 ] ; then echo " Timeout, aborting." ; exit 1 ; fi ; RETRIES=$$(($${RETRIES}+1)) ; done
-	cat ./tmp/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./tmp/clair/%.json" -l ./tmp/clair/clair.log %'; \
+	cat ./docker-image/build-*.tags | xargs -I % sh -c 'clair-scanner --ip 172.17.0.1 -r "./clair/%.json" -l ./clair/clair.log %'; \
 	XARGS_EXIT=$$?; \
-	if [ $${XARGS_EXIT} -eq 123 ]; then find ./tmp/clair/wyrihaximusnet -type f | sed 's/^/-Fjson=@/' | xargs -d'\n' curl -X POST ${WALLE_REPORT_URL} -F channel=team_oz -F buildUrl=https://circleci.com/gh/wyrihaximusnet/docker-php/${CIRCLE_BUILD_NUM}#artifacts/containers/0; else exit $${XARGS_EXIT}; fi
\ No newline at end of file
+	if [ $${XARGS_EXIT} -eq 123 ]; then find ./clair/wyrihaximusnet -type f | sed 's/^/-Fjson=@/' | xargs -d'\n' curl -X POST ${WALLE_REPORT_URL} -F channel=team_oz -F buildUrl=https://circleci.com/gh/wyrihaximusnet/docker-php/${CIRCLE_BUILD_NUM}#artifacts/containers/0; else exit $${XARGS_EXIT}; fi
```

Comment:
reference to circleci here
unfortunately there is no current way to get the github actions build number

Reply:
There is, you'd need `jq` but you can het it from the event file. Let me fix that and tag you in the PR :)

Reply:
hmm that must be new! thanks

Reply:
any luck? still not available AFAICS
https://github.community/t5/GitHub-Actions/Get-Runs-Url/m-p/38506

Reply:
Not luck yet, been trying to get a json string of the github variable in the runner but no luck so far